### PR TITLE
Mimecast TTP

### PR DESCRIPTION
**Pull request by nipun-crestdatasystem**
This is a draft PR to run the validation checks for Mimecast TTP.

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Thanks @nipun-crestdatasystem for raising this PR, we will review by 30 Aug 2023

**Comment by v-rbajaj** [comment]
Hi @nipun-crestdatasystem, the folders inside Data connector folder are incorrectly placed, please make following changes : - 

- Move Parsers folder to \Solutions\MimecastTTP\

- Remove the logo folder and move the logo to https://github.com/Azure/Azure-Sentinel/tree/master/Logos
 (you would also need to change the path of the logo in the Solutions/MimecastTTP/Data/Solution_MimecastTTP.json file)

File: Solutions/MimecastTTP/Analytic Rules/MimecastTTPAttachment.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+id: aa75944c-a663-4901-969e-7b55bfa49a73
+name: Mimecast Targeted Threat Protection - Attachment Protect
+description: Mimecast Targeted Threat Protection - Attachment Protect
```
Comment (v-rbajaj) [inline Solutions/MimecastTTP/Analytic Rules/MimecastTTPAttachment.yaml:3]: Description needs to be descriptive in all the three analytical rules.

File: Solutions/MimecastTTP/Analytic Rules/MimecastTTPAttachment.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+query: MimecastTTPAttachment_CL| where result_s <> "safe";
+queryFrequency: 5m
+queryPeriod: 15m
+triggerOperator: gt
+triggerThreshold: 0
+suppressionDuration: 5h
+suppressionEnabled: false
+tactics:
+- InitialAccess
+- Discovery
```
Comment (v-rbajaj) [inline Solutions/MimecastTTP/Analytic Rules/MimecastTTPAttachment.yaml:19]: Please add relevantTechniques, Relevant [MITRE Techniques](https://attack.mitre.org/versions/v11/matrices/enterprise/) ID.

This needs to be added in all the analytical rules of the PR.

Example :- https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/Azure%20Active%20Directory/Analytic%20Rules/FailedLogonToAzurePortal.yaml

File: Solutions/MimecastTTP/SolutionMetadata.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "publisherId": "azuresentinel",
```
Comment (v-rbajaj) [inline Solutions/MimecastTTP/SolutionMetadata.json:2]: The tier is mentioned as Partner. So the publisherID needs to be changed according to the publisherID of the partner.

**Comment by v-rbajaj** [comment]
Also apart from the above mentioned changes, please make the following changes :

Please do add release notes, please follow this link -https://github.com/Azure/Azure-Sentinel/blob/master/Solutions/ReleaseNotesGuidance.md

Please add workbook metadata here - https://github.com/Azure/Azure-Sentinel/blob/master/Tools/Create-Azure-Sentinel-Solution/V2/WorkbookMetadata/WorkbooksMetadata.json

After all these changes, repackage the solution.

**Comment by nipun-crestdatasystem** [comment]
@v-rbajaj thanks for the review we are working on the changes and we will get back to you as soon as possible.

**Comment by v-rbajaj** [comment]
Hi @nipun-crestdatasystem, please resolve merge conflicts and update this branch from master.

File: Solutions/MimecastTTP/SolutionMetadata.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+    "publisherId": "mimecast",
+    "offerId": "mimecastttp",
```
Comment (v-rbajaj) [inline Solutions/MimecastTTP/SolutionMetadata.json:3]: Please update the offer ID as per this document 
https://github.com/Azure/Azure-Sentinel/tree/master/Solutions#author-and-publish-a-solution-offer

**Comment by v-rbajaj** [comment]
Hi @nipun-crestdatasystem, please resolve merge conflicts and will validate changes by 15 Sep 2023.

**Review by v-rbajaj — APPROVED** [review]

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**